bundles/org.openhab.binding.plclogo/src/main/java/org/openhab/binding/plclogo/internal/handler/PLCAnalogHandler.java

```diff
@@ -256,13 +256,13 @@ protected void doInitialization() {
 
             String type = config.get().getChannelType();
             for (int i = 0; i < getNumberOfChannels(); i++) {
-                String name = kind + String.valueOf(i + 1);
+                String name = kind + i + 1;
```

Comment:
Sorry about the noise, but from my point of view, this will break the binding. For `i = 0`, the name will not be `AI1`, but `AI01`. Miss i something?

Reply:
Good catch!